client/main.html

```diff
@@ -4,4 +4,5 @@
 
 <body>
   <div id="render-target"></div>
+  <div class="footer"><a href="main.html" class="link1">About us</a>&nbsp; | &nbsp;<a href="main.html" class="link1">Contact us</a></div>
```

Comment:
Let's add this to the AppFrame instead.  We can create a Footer component, similar to the Navbar component, and then render our app like this:
```
  render() {
    return(
      <div className="container-fluid">
          <Navbar />
        <div className="content">
          {this.props.children}
        </div>
          <Footer />
      </div>
    );
  }
```